e2e/tests/interchain_accounts/base_test.go

```diff
@@ -81,7 +82,7 @@ func (s *InterchainAccountsTestSuite) TestMsgSendTx_SuccessfulTransfer() {
 			// fund the host account so it has some $$ to send
 			err := chainB.SendFunds(ctx, interchaintest.FaucetAccountKeyName, ibc.WalletAmount{
 				Address: hostAccount,
-				Amount:  testvalues.StartingTokenAmount,
+				Amount:  sdkmath.NewInt(testvalues.StartingTokenAmount),
```

Comment:
ah, this is w.r.t the issue in #4396 yea?

Reply:
sure is :)

to replicate this pr, you'd check out main, then:

```bash
cd e2e
go get github.com/strangelove-ventures/interchaintest/v7
go mod tidy
```

Then you basically just follow along you'll see the errors pop up in your editor.

Reply:
if at first you don't succeed, try, try again :rocket: